crates/scheduler/examples/parallel.rs

```diff
@@ -16,7 +16,7 @@ fn main() -> Result<(), Report> {
         .add_system(read_write_many);
 
     for i in 0..20 {
-        let entity = app.create_entity();
+        let entity = app.create_entity().unwrap();
```

Comment:
while it's fine to use `.unwrap()` in tests (since they should fail with a panic if something's wrong), our example code should be written much more nicely and be exemplary in its handling of errors (i.e. propagating them instead of panicking immediately)
```suggestion
        let entity = app.create_entity()?;
```

Reply:
> while it's fine to use .unwrap() in tests (since they should fail with a panic if something's wrong)

Or what do you mean?

Reply:
I thought this was in the ecs/lib test, my bad